torch_em/util/image.py

```diff
@@ -35,6 +37,9 @@ def load_image(image_path, memmap=True):
         return tifffile.memmap(image_path, mode="r")
     elif tifffile is not None and os.path.splitext(image_path)[1].lower() in (".tiff", ".tif"):
         return tifffile.imread(image_path)
+    elif os.path.splitext(image_path)[1].lower() in (".nrrd"):
```

Comment:
I don't think this works / does what is expected, please double check: `(".nrrd")` is the same as `".nrrd"` and then the check is against the elements in the string.
You probably want `(".nrrd",)` instead.